restapi/src/main/java/io/stargate/web/resources/v2/schemas/KeyspacesResource.java

```diff
@@ -75,11 +103,35 @@ public Response listAll(
 
   @Timed
   @GET
+  @ApiOperation(
+      value = "get a keyspace",
```

Comment:
Lower case here, and proper case in other places. This is for several other operations.